src/greentea/gtea/greentea_cli.py

```diff
@@ -31,14 +36,17 @@ def get_local_host_tests_dir(path):
 def create_filtered_test_list(
     ctest_test_list, test_by_names, skip_test, test_spec=None
 ):
-    """! Filters test case list (filtered with switch -n) and return filtered list.
-    @ctest_test_list List of tests, derived from test specification
-    @test_by_names Command line switch -n <test_by_names>
-    @skip_test Command line switch -i <skip_test>
-    @param test_spec Test specification object loaded with --test-spec switch
-    @return
-    """
+    """Create a filtered test case list.
+
+    Args:
+        ctest_test_list: List of tests, derived from test specification.
```

Comment:
`ctest_test_list` is an outdated name. The tool doesn't use CTestTestFile.cmake anymore. Let's just change this to `test_list`?

Reply:
I've updated to remove any `ctest_` prefixes, as well as code that was previously used when using CTestTestFile.cmake.